conftest.py

```diff
@@ -18,6 +18,21 @@
 
 
 def pytest_addoption(parser):
+    #Required for all tests.
+    parser.addoption('--pinned-hostname', action='store', type=str,
+                     help='agent host to pin storage volumes to', required=True)
+   
+    #Needed for scale-testing, missing values will prompt scale test to error out.
+    parser.addoption('--datadog-api-key', action='store', type=str, default=None,
```

Comment:
should these fields (datadog-api-key, datadog-plugin-hostname)  be marked with required=true?

Reply:
Currently the metrics plugin is only installed for test_soak, it not used in test_scale, hence these two fields were not flagged as required.